src/controllers/pages/student.js

```diff
@@ -13,11 +13,23 @@ export default class StudentController{
     }
 
     static cancelCheckout() {
-        Dispatcher.handleAction('CLEAR_ITEMS');
+        Dispatcher.handleAction('CLEAR_CONTENTS');
```

Comment:
it does?  What does the cart contain: contents.  It clears the contents of the cart.  The other names are fine too, but there's nothing wrong with this one.